config/routes.rb

```diff
@@ -3,9 +3,6 @@
 Frontend::Application.routes.draw do
   get "/homepage" => redirect("/")
 
-  get "/search" => "search#index", as: :search
-  get "/search/opensearch" => "search#opensearch"
```

Comment:
Has this endpoint been moved to finder-frontend? Just checking because I don't recognise it.

Reply:
Yes, this is now being render from finder-frontend